editoast/src/generated_data/utils.rs

```diff
@@ -22,29 +19,29 @@ impl<'a> InvolvedObjects<'a> {
     /// For example if you have an operation that modify a Track Section, and you call this function with `ObjectType::Signal` as `obj_type`,
     /// You'll have the set of signal ids that are linked to the track section that has been modified.
     pub fn from_operations(
-        operations: &'a [OperationResult],
+        operations: &'a [CacheOperation],
         infra_cache: &'a InfraCache,
         obj_type: ObjectType,
     ) -> Self {
         let mut res = Self::default();
         for op in operations {
             match op {
-                OperationResult::Create(railjson) | OperationResult::Update(railjson)
+                CacheOperation::Create(railjson) | CacheOperation::Update(railjson)
```

Comment:
```suggestion
                CacheOperation::Create(object_cache) | CacheOperation::Update(object_cache)
```

Reply:
I'm not bothered by the current solution. We know that we'll certainly have to change what the apply function returns.

Reply:
Done in https://github.com/osrd-project/osrd/compare/7fc57c1adac6bce87ef213e4588edda0bebe2648..e4f6220d1cfa01abcf41a1e2ca1ff9d7c6d99c02.